sns_ik_lib/matlab/TEST_acc_ik_solvers.m

```diff
@@ -0,0 +1,34 @@
+% This script sets up an runs a set of unit tests on the acceleration IK solvers.
+%
+%
+% LICENSE:
```

Comment:
License is redundant, you can remove that line, and un-indent the rest of the license

Reply:
fixed